Address [I18N-ACTION-1178] by merging useful bits of i18n-html-tech-lang

aphillips opened the pull request:
Mainly this is the "outright theft" of section 2 in our old document found [here](https://www.w3.org/TR/i18n-html-tech-lang/#ri20040808.100519373). 

- Added an introductory paragraph
- Added a new example of `lang`
- Fixed the Respec "error"/warning about CLDR with a `lint-ignore`

When reviewing, consider whether we should provide any mustard and whether this is located in the right place in the document (vs. after the materials about locale and I18N)


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/ltli/pull/34.html" title="Last updated on Jun 11, 2026, 5:18 PM UTC (e1a20e9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/ltli/34/27cd8a2...aphillips:e1a20e9.html" title="Last updated on Jun 11, 2026, 5:18 PM UTC (e1a20e9)">Diff</a>

aphillips committed:
Address [I18N-ACTION-1178] by merging useful bits of i18n-html-tech-lang

Merged definitions of intended audience and text-processing language with minor edits.

Addition of a new example.

aphillips committed:
Merge branch 'w3c:gh-pages' into gh-pages

xfq commented on index.html:
```diff
@@ -189,8 +189,53 @@ <h3>Languages and Language Tags</h3>
+
+        <p>There are also pages where the navigational information, including the page title, is in one language but the real content of the page is in another. While this is not necessarily good practice, it doesn't change the fact that the language of the intended audience is usually that of the content, regardless of the language at the top of the document source.</p>
+
+        <p>Metadata about the language of the intended audience is usually best declared outside the document, such as in the HTTP <kbd>Content-Language</kbd> header.</p>
```
Why is this `kbd` instead of `code` (same below)?

r12a replied:
I think it should actually be `<span class="kw" translate="no">`

aphillips replied:
Done

aphillips committed:
Address @r12a's comments

Change `kbd` to `span class=kw`

aphillips committed:
Move new material per telecon, add `kw` style

Move the use cases section down in the document.

Add a `.kw` style to `local.css`

xfq commented:
There seems to be some overlap between this section and the [Types of language declaration](https://www.w3.org/International/questions/qa-text-processing-vs-metadata) article. What is the difference?

aphillips commented:
@xfq Yes, there's overlap. Note that LTLI is in the TR space, while the article is "just" an article. A question would be whether we should reference the article from LTLI (to cut down on overall repetition)

aphillips committed:
Merge branch 'w3c:gh-pages' into gh-pages
